solutions/observability/logs/streams/management/extract.md

```diff
@@ -0,0 +1,155 @@
+
+The UI also shows indexing problems, such as mapping conflicts, so you can address them before applying changes.
+
+:::{note}
+Applied changes aren't retroactive and only affect *future data ingested*.
+:::
+
+## Add a processor [streams-add-processors]
+
+Streams uses {{es}} ingest pipelines to process your data. Ingest pipelines are made up of processors that transform your data.
+
+To add a processor:
+
+1. Select **Add processor** to open a list of supported processors.
+1. Select a processor from the list:
+    - [Date](./extract/date.md)
+    - [Dissect](./extract/dissect.md)
+    - [Grok](./extract/grok.md)
+    - [Key-Value (KV)](./extract/key-value.md)
+    - GeoIP
+    - Rename
+    - Set
+    - URL Decode
+1. Select **Add Processor** to save the processor.
+
+:::{note}
+Editing processors with JSON is planned for a future release. More processors may be added over time.
+:::
+
+### Add conditions to processors [streams-add-processor-conditions]
+
+You can provide a condition for each processor under **Optional fields**. Conditions are boolean expressions that are evaluated for each document. Provide a field, a value, and a comparator.
+Processors support these comparators:
+- equals
+- not equals
+- less than
+- less than or equals
+- greater than
+- greater than or equals
+- contains
+- starts with
+- ends with
+- exists
+- not exists
+
+### Ignore failures [streams-ignore-failures]
+
+Turn on **Ignore failure** to ignore the processor if it fails. This is useful if you want to continue processing the document even if the processor fails.
+
+### Ignore missing fields [streams-ignore-missing-fields]
+
+Turn on **Ignore missing fields** to ignore the processor if the field is not present. This is useful if you want to continue processing the document even if the field is not present.
+
+### Preview changes [streams-preview-changes]
+
+Under **Processors for field extraction**, set pipeline processors to modify your documents. **Data preview** shows you a preview of the results, with additional filtering options depending on the outcome of the simulation.
+
+When you add or edit processors, the **Data preview** updates automatically.
+
+:::{note}
+To avoid unexpected results, focus on adding processors rather than removing or reordering existing processors.
+:::
+
+**Data preview** loads 100 documents from your existing data and runs your changes using them.
+For any newly added processors, this simulation is reliable. You can save individual processors during the preview, and even reorder them.
+Selecting 'Save changes' applies your changes to the data stream.
+
+If you edit the stream again, note the following:
+- Adding more processors to the end of the list will work as expected.
+- Changing existing processors or re-ordering them may cause unexpected results. Because the the pipeline may have already processed the documents used for sampling, the UI cannot accurately simulate changes to existing data.
```

Comment:
```suggestion
- Changing existing processors or re-ordering them may cause unexpected results. Because the pipeline may have already processed the documents used for sampling, the UI cannot accurately simulate changes to existing data.
```